doc/tags/for.rst

```diff
@@ -58,15 +58,17 @@ Inside of a ``for`` loop block you can access some special variables:
 ===================== ========================================================================
 Variable              Description
 ===================== ========================================================================
-``loop.index``        The current iteration of the loop. (1 indexed)
-``loop.index0``       The current iteration of the loop. (0 indexed)
+``loop.index``        The current iteration of the loop (1 indexed)
+``loop.index0``       The current iteration of the loop (0 indexed)
 ``loop.revindex``     The number of iterations from the end of the loop (1 indexed)
 ``loop.revindex0``    The number of iterations from the end of the loop (0 indexed)
 ``loop.first``        True if first iteration
 ``loop.last``         True if last iteration
 ``loop.length``       The number of items in the sequence
 ``loop.parent``       The parent context
 ``loop.cycle``        Cycle over a sequence of values
```

Comment:
Should this one be removed this list, as it's not a variable but a function ? (#nitpicking)

(the only one from this list i believe)

Reply:
(i tried to point a the `loop.cycle` line ... and failed miserably 😅 )

Reply:
See #4159